[E 2025-04-23 19:11:22,605 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-23 19:22:12,292 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-23 19:29:21,519 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-23 19:31:52,709 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 12:12:07,695 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 17:17:22,457 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 17:17:37,481 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:461:cancel_order] StatusCode: 400, Detail: Please run update_status first to get trade details.
[E 2025-04-24 17:17:37,481 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 400}, 'response': {'detail': 'Please run update_status first to get trade details.'}}
[E 2025-04-24 17:49:21,199 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 17:49:36,237 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:461:cancel_order] StatusCode: 400, Detail: Please run update_status first to get trade details.
[E 2025-04-24 17:49:36,237 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\utils.py:49:raise_resp_error] {'status': {'status_code': 400}, 'response': {'detail': 'Please run update_status first to get trade details.'}}
[E 2025-04-24 19:27:55,106 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 19:28:16,319 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 19:28:35,903 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 19:28:36,460 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 19:28:36,891 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-24 19:28:37,379 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:423:place_order] {'status': {'status_code': 406}, 'response': {'detail': 'Order price error.'}}
[E 2025-04-25 10:38:58,644 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 14:57:43,809 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:07:09,873 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:16:20,672 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:29:40,785 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:38:52,455 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:55:20,529 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 15:58:32,668 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-28 16:12:38,978 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 10:33:23,112 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 10:33:43,678 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 10:33:50,449 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('_ssl.c:983: The handshake operation timed out'))
[E 2025-04-29 11:06:12,329 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:06:32,866 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:06:38,374 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:12:48,761 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:23:38,737 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:26:21,470 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:54:00,864 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-29 11:54:21,412 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 09:45:03,186 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 09:45:03,690 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 09:45:04,198 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 09:45:04,700 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 09:45:05,201 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 09:45:05,703 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(gaierror(11001, 'getaddrinfo failed'))
[E 2025-04-30 13:30:36,459 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:30:41,985 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:30:47,501 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:30:53,015 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:30:58,536 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:31:04,050 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:31:24,410 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:31:29,919 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:31:35,429 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
[E 2025-04-30 13:31:40,952 C:\Users\charl\vnpy_inter_shioaji\.venv\Lib\site-packages\shioaji\shioaji.py:231:_setup_solace] RequestException URLError(TimeoutError('timed out'))
